dev-tools/packer/docker/fpm-image/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM ubuntu:16.04
+# FROM ubuntu:16.04
```

Comment:
did this `#` sneak in or is that intentional?

Reply:
I really need coffee

Reply:
Sorry for the waste of time.

Reply:
not at all, thanks for doing the backport